tests/integration/test.basics.js

```diff
@@ -1106,5 +1106,21 @@ adapters.forEach(function (adapter) {
         });
       });
     }
+
+    if (typeof process !== 'undefined' && !process.browser) {
+      it('#5471 PouchDB.plugin() should throw error if passed wrong type or empty object', function () {
+        var db = new PouchDB(dbs.name);
+        PouchDB.plugin('pouchdb-adapter-memory');
```

Comment:
I think the error should be thrown at this point. If the test gets past this line, it should fail.